src/normalize/normalize.py

```diff
@@ -163,29 +156,56 @@ def parse_files(source_prefix, destination_prefix, start_date, state_file):
                 LOGGER.warning(f"No data found in partition: {date_partition} - is this expected?")
             LOGGER.info(f"Updating last processed timestamp to maximum file timestamp: {max_timestamp}")
             update_last_processed_timestamp(s3, state_bucket, state_key, max_timestamp)
-        cur_processing += datetime.timedelta(days=1)
+        cur_processing += dt.timedelta(days=1)
+
+
+def validate_date(ctx, param, value):
+    if value is None:  # If no input is provided, use today's date
+        return str(dt.date.today())
+    try:
+        # Attempt to parse the date with the expected format
+        return str(dt.datetime.strptime(value, '%Y%m%d').date())
+    except ValueError:
+        # Raise a BadParameter error if the format is incorrect
+        raise click.BadParameter('Date must be in YYYYMMDD format.')
 
 
+@click.command()
+@click.option("-f", "--feed_id", required=True, type=str, help="feed ID to be scraped")
```

Comment:
is there a reason this should be a separate cli option instead of passing straight through the config?

Reply:
Resolved as we went thru in the call